src/components/layout/Layout.module.scss.d.ts

```diff
@@ -0,0 +1,10 @@
+export declare const aminoLayout: string;
+export declare const content: string;
+export declare const contentGrid: string;
+export declare const footer: string;
+export declare const header: string;
+export declare const searchInputWrapper: string;
+export declare const sidebarContent: string;
+export declare const styledLogoSidebar: string;
+export declare const styledSearchInput: string;
+export declare const styledSidebar: string;
```

Comment:
Is this a manual process to create this file?

Reply:
No, I includes the command to generate types for scss file type in the `pnpm dev` and `pnpm build`. Those two commands are already part of the developing process, so the process would not change at all, it would just magically work.